cpu/cc2538/clock.c

```diff
@@ -61,8 +61,20 @@
 
 #include <stdint.h>
 /*---------------------------------------------------------------------------*/
-#define RTIMER_CLOCK_TICK_RATIO (RTIMER_SECOND / CLOCK_SECOND)
-#define RELOAD_VALUE (125000 - 1)  /** Fire 128 times / sec */
+#define RTIMER_CLOCK_TICK_RATIO			(RTIMER_SECOND / CLOCK_SECOND)
+
+/* Prescaler for GPT0:Timer A used for clock_delay_usec(). */
+#if SYS_CTRL_SYS_CLOCK < SYS_CTRL_1MHZ
+#error System clock speeds below 1MHz are not supported
+#endif
+#define PRESCALER_VALUE					(SYS_CTRL_SYS_CLOCK / SYS_CTRL_1MHZ - 1)
```

Comment:
The assigned value is too far on the right, exceeding the 80-char limit. Moreover, only 2 of the 3 definitions are aligned.